sdks/python/apache_beam/runners/dataflow/dataflow_runner.py

```diff
@@ -566,6 +572,17 @@ def run_pipeline(self, pipeline, options):
     result.metric_results = self._metrics
     return result
 
+  def _maybe_add_unified_worker_missing_options(self, options):
+    # set default beam_fn_api and use_unified_worker experiment if
+    # 'use_runner_v2' experiment flag exists, no-op otherwise.
+    experiments = options.view_as(DebugOptions).experiments or []
```

Comment:
Debug options have has convenience method for lookup and adding experiments. We can use those instead of raw list.